src/latex/agda-latex-macros.sty

```diff
@@ -273,22 +273,23 @@
 \newcommand{\getStakeDist}{\AgdaFunction{getStakeDist}\xspace}
 \newcommand{\gid}{\AgdaField{gid}\xspace}
 \newcommand{\GovAction}{\AgdaDatatype{GovAction}\xspace}
+\newcommand{\govActionDeposit}{\AgdaField{govActionDeposit}\xspace}
 \newcommand{\GovActionID}{\AgdaDatatype{GovActionID}\xspace}
 \newcommand{\GovActionIDFunction}{\AgdaFunction{GovActionID}\xspace}
-\newcommand{\GovActionState}{\AgdaRecord{GovActionState}\xspace}
-\newcommand{\govActionDeposit}{\AgdaField{govActionDeposit}\xspace}
-\newcommand{\GovernanceGroup}{\AgdaInductiveConstructor{GovernanceGroup}\xspace}
 \newcommand{\govActionLifetime}{\AgdaField{govActionLifetime}\xspace}
+\newcommand{\govActionPriority}{\AgdaFunction{govActionPriority}\xspace}
```

Comment:
Only this line is new, but this section was getting out of order.  It's useful to keep them in alphabetical order as this makes it much easier to spot duplicates and quickly check whether a macro for a given Agda keyword has been defined yet.